test/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
```

Comment:
Is this file needed now that we're filtering the deprecation warnings in the example notebook?

Reply:
It looks like not needed - I removed it.